samples/net/lwm2m_client/README.rst

```diff
@@ -80,12 +80,12 @@ Build the lwm2m-client sample application for QEMU like this:
 .. zephyr-app-commands::
    :zephyr-app: samples/net/lwm2m_client
    :host-os: unix
-   :board: qemu_x86
+   :board: qemu_cortex_m3
```

Comment:
I'm not also 100% convinced about changing the default platform, `qemu_x86` is defacto standard used in many samples.

Reply:
+1 to this, I would still keep qemu_x86 as a default because it gets tested much more compared to Arm Ethernet driver so the qemu_x86 could work more reliable (hopefully)

Reply:
Returned the default back to QEMU_X86